modules/imgimport/test/test.cpp

```diff
@@ -48,7 +49,8 @@ BOOST_AUTO_TEST_CASE(DecklinkVideoSource){
         BOOST_ERROR("Invalid number of arguments");
     }
 
-    DeckLinkImport * v = new DeckLinkImport(NULL);
+    Camera* cam = new Camera(cv::Size(),cv::Size2d(),cv::Mat(), cv::Mat(), cv::Mat());
```

Comment:
There's a Camera::TestCamera() function which can be used for this. Admittedly it's set up for the gopro, but should give valid images. I have no idea what the above camera would do to an image.